terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -642,6 +632,7 @@ def remove_inconsequential_statements(statements: List[JSON]) -> List[JSON]:
             },
             "gitlab": {
                 "name": "azul-gitlab",
+                "tags": {"component": "azul-gitlab"},
```

Comment:
Why doesn't this call `tags()`?

Reply:
In cases where the name of a resource doesn't follow the qualified resource name format, the `component` tag can't be inferred, so I specify it manually. For consistency, I'll change this to `tags`